tests/websockets_SUITE.erl

```diff
@@ -67,7 +67,9 @@ chat_msg(Config) ->
     escalus:story(Config, [{alice, 1}, {geralt, 1}, {oldie, 1}], fun(Alice, Geralt, Oldie) ->
 
         escalus_client:send(Alice, escalus_stanza:chat_to(Geralt, <<"Hi!">>)),
-        escalus:assert(is_chat_message, [<<"Hi!">>], escalus_client:wait_for_stanza(Geralt)),
+        FromAlice = escalus_client:wait_for_stanza(Geralt),
+        escalus:assert(is_chat_message, [<<"Hi!">>], FromAlice),
+        escalus:assert(has_ns, [<<"jabber:client">>], FromAlice),
```

Comment:
You can also add check for absence of the "jabber:client"  namespace in the message received by oldie.  https://github.com/esl/ejabberd_tests/pull/106/files#diff-571321649bb18acb620755061dfc3fe0L76